src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs

```diff
@@ -650,6 +650,16 @@ public enum NuGetLogCode
         /// </summary>
         NU5037 = 5037,
 
+        /// <summary>
+        /// Waring_IconURLDeprecationMessage
+        /// </summary>
+        NU5048 = 5048,
+
+        /// <summary>
+        /// Waring_IconUrlAndIconDeprecationMessage
```

Comment:
```suggestion
        /// Warning_IconUrlAndIconDeprecationMessage
```

Reply:
I will change to its respective string resource.